packages/fast-components-react-base/src/typography/typography.props.ts

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { IManagedClasses, ITypographyClassNameContract } from "@microsoft/fast-components-class-name-contracts";
+
+export enum TypographyTag {
+    caption = "caption",
+    h1 = "h1",
+    h2 = "h2",
+    h3 = "h3",
+    h4 = "h4",
+    h5 = "h5",
+    h6 = "h6",
+    p = "p",
+    span = "span"
+}
+
+export enum TypeLevel {
+    _1 = 1,
```

Comment:
I assume the _ is needed to use a number as a type?

Reply:
correct - can't start with a number

Reply:
cool